start_dev_db.sh

```diff
@@ -1,25 +0,0 @@
-#!/usr/bin/env bash
-
-SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-cd $SCRIPT_DIR
-
-echo "dropping old cme_* artifacts"
-echo "----------------------------------------"
-docker container rm cme_mongodb
-docker network rm cme_internal
-docker volume rm cme_db-volume
-docker volume rm cme_configdb-volume
-echo ""
-
-echo "starting services"
-echo "----------------------------------------"
-docker-compose -f ./docker-compose.dev.yml --env-file ./.env.dev up 
-echo ""
-
-echo "dropping cme_* services"
-echo "----------------------------------------"
-docker container rm cme_mongodb
-docker network rm cme_internal
-docker volume rm cme_db-volume
-docker volume rm cme_configdb-volume
-
```

Comment:
Is there a reason why we drop this script? Otherwise, running the dev db would require calling docker-compose manually as well as dropping the volumes and so on manually